NEWS.md

```diff
@@ -3,6 +3,40 @@ NEWS
 
 Here we have a summary of the major changes by release. See the git commit history or `ChangeLog` for more fine-grained changes.
 
+Version 2.1.1
+-------------
+
+2025-01-03
+
+- More checks of potentially NULL buffers. More check of the result of malloc/calloc.
+- Enforce the use of non widestring (A suffixed) calls when we pass char* parameters. (Pete Batard)
+- Use widestring API calls unless otherwise specified. So make sure we
+  char* parameters.  Remove a warning about the declaration of is_cdrom_aspi() and about GetVersion() being deprecated. (Pete Batard)
+- Updates for compiling on MSVC. (Pete Batard)
+- Move to github and add CI checks. (rocky for move Pete Batard for CI)
+- Count empty fields as tracks. (John Ernberg)
+- Add some validity checks to enhance security. (Mansour Gashasbi)
+- Add support for ISO9660 multi extent files. (Pete Batard)
+- Fix Recognition of multi-extent in ISO9660 when Joliet is present. (Thomas Schmitt)
+- Use getmntent/setmntent for reading mounts. (Miguel Borges de Freitas)
+- Use GNU/Linux new ioctl on kernel v5.16 or newer. (Lukas Prediger)
+- Use "%s"-style format in cdda-player.c: to make it catch cases.
+  when user input is used in place of format. (Sergei Trofimovich)
+- Remove some memory leaks in C++ code. (Thomas Schmitt)
+- Allow for  DO_NOT_WANT_COMPATIBILITY macro in config.h to disable APIs that are being retired.
+- Fix win32 implementation of .get_track_msf() for CD with first track number > 1 (Thomas Schmitt)
+- Fix testing on Windows and remove compilation warnings. (Pete Batard)
+- Add Rock Ridge deep directory support. (Pete Batard)
+- Fix and clean up various Rock Ridge issues and adjust tests. (Pete Batard)
+- Fix double reporting of sizes in cd-info.c. (Pete Batard)
+- CD-Text character set interpretation more tolerant of bad input. Savannah bug 53929.
+- Remove homegrown boolean type in favor of <stdbool.h>. (Thomas Schmitt)
+- Improve pkg-config configuration detection (Lars Wendler)
+- Fix crash reading CD TOC on macOS Ventura. (Robert Kausch)
+- Update freedb references to GnuDB. (Robert Kausch)
+- Fix charset check in Windows cdio_charset_from_utf8 implementation. (Robert Kausch)
+- Add support for reading CD-Text using the Win32 IOCTL driver (Robert Kausch)
+
```

Comment:
`- Add support for reading CD-Text on OSX`

Should be mentioned here.

Reply:
Thanks - fixed now.